src/pages/gen2/build-a-backend/storage/index.mdx

```diff
@@ -11,44 +11,242 @@ export function getStaticProps(context) {
   };
 }
 
-<Callout info>
+<Callout warning>
 
-**Coming soon:** The Storage experience for Amplify (Gen 2) is still under development. You can add Amazon S3 to your Amplify (Gen 2) project with the AWS CDK.
+**Under active development:** The Storage experience for Amplify (Gen 2) is under active development. The experience may change between versions of `@aws-amplify/backend`. Try it out and provide feedback at https://github.com/aws-amplify/amplify-backend/issues/new/choose
 
 </Callout>
 
-To use Storage today, you must use the [AWS Cloud Development Kit (CDK)](https://docs.aws.amazon.com/cdk/), which is installed by default as part of the [`create-amplify`](https://www.npmjs.com/package/create-amplify) workflow. 
+Adding storage to your Amplify backend enables uploading and downloading files. To get started using storage, create a file `amplify/storage/resource.ts`. Paste the following content into the file.
 
-To get started, modify your project's `backend.ts` file to create an Amazon S3 bucket and grant Amplify-generated resources access to read and write:
+```ts
+// amplify/storage/resource.ts
+import { defineStorage } from '@aws-amplify/backend';
 
-```ts title="amplify/backend.ts"
+export const storage = defineStorage({
+  name: 'myProjectFiles'
+});
+```
+
+Then include storage in your backend definition.
+
+```ts
 // amplify/backend.ts
-import * as s3 from 'aws-cdk-lib/aws-s3';
 import { defineBackend } from '@aws-amplify/backend';
-import { auth } from './auth/resource.js';
-import { data } from './data/resource.js';
+import { storage } from './storage/resource';
+
+defineBackend({
+  storage
+});
+```
+
+Now when you run `npx amplify sandbox` or deploy your app on Amplify, it will configure AWS resource for file upload and download.
```

Comment:
also add docs on how to actually read the data from the client-side.

Reply:
I added links to the existing frontend storage upload/download docs